src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -13,6 +13,7 @@
     isFullscreen
     :showClose="false"
     :visible.sync="isVisible"
+    backdropBackgroundColorOverride="rgba(0, 0, 0, 0.7)"
```

Comment:
This is not light/dark mode aware. Is that intended?

Reply:
I believe that the modal background is always in black. For dark and light mode. it's like that in `main` too

Reply:
Let's change the background to color-fill-secondary to add a bit more contrast in dark mode. 
<img width="2055" alt="Screenshot 2022-11-28 at 10 26 58 AM" src="https://user-images.githubusercontent.com/11503099/204352861-c5adad61-4d31-4795-9c66-da2b3ede7be1.png">

Reply:
> Let's change the background to color-fill-secondary to add a bit more contrast in dark mode.

Done! @biamx3